### PR TITLE
[KOGITO-984] Remove e2e tests

### PR DESCRIPTION
**Pull request by sutaakar**
https://issues.redhat.com/browse/KOGITO-984

Signed-off-by: Karel Suta <ksuta@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -813,52 +813,6 @@ To verify the operator status, run the following command:
 $ oc describe operatorsource.operators.coreos.com/kogito-operator -n openshift-marketplace
 ```
 
-### Running End-to-End (E2E) tests
```
Comment (radtriste) [inline README.md]: don't forget to update the index :)

**Comment by sutaakar** [comment]
@radtriste Done.
Also cleaned up vendor folder.

**Comment by ricardozanini** [comment]
One less thing to take care :)

**Comment by ricardozanini** [comment]
@spolti wanna take a look?

**Pull request merged**